frontend/src/app/home/page.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export default function HomePage() {
+  return (
+    <div style={{ padding: '40px', textAlign: 'center' }}>
```

Comment:
Why do we have inline styles here? Is there a reason you're not setting a class?